ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py

```diff
@@ -418,38 +437,46 @@ def patch_column_tags(
 
         return patched_entity
 
-    def patch_column_tag(
+    @deprecated(
+        message="Use metadata.patch_column_descriptions instead", release="1.3.1"
+    )
+    def patch_column_description(
```

Comment:
Idea was to deprecate it for 1.3.1 and we have new method `patch_column_tags` which will replace it. The changes for this are already done by @pmbrull .

But since we said we will do for 1.3.1 I'll add the method back

Reply:
Okay I see the patch_column_tag method is still there, got confused with the git diff